app-backup/borgbackup/borgbackup-1.2.0.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Deduplicating backup program with compression and authenticated encryption"
+DESCRIPTION="BorgBackup (short: Borg) is a deduplicating backup program.
+Optionally, it supports compression and authenticated encryption.
+
+The main goal of Borg is to provide an efficient and secure way to backup data.
+The data deduplication technique used makes Borg suitable for daily backups
+since only changes are stored.
+The authenticated encryption technique makes it suitable for backups to not
+fully trusted targets."
+HOMEPAGE="https://github.com/borgbackup/borg"
+COPYRIGHT="201-2021 Mathias Vogelgesang"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="7659322a078dd3705f89345d0b18b4cdb2d7fee1b65391b0596cb75a19cac64f"
+SOURCE_FILENAME="borgbackup-$portVersion.tar.gz"
+SOURCE_DIR="borg-$portVersion"
+SOURCE_URI_2="https://github.com/hynek/argon2-cffi/archive/refs/tags/21.3.0.tar.gz"
+CHECKSUM_SHA256_2="1f5cdc94200eccdf8c63c1cc5c0c72e38076d8e81968218b0ec0573fd72b2ba4"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	$portName = $portVersion
+	cmd:borg
+	cmd:borgfs
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	argon2_cffi_bindings${secondaryArchSuffix}_python39
+	cffi${secondaryArchSuffix}_python39
+	msgpack_python39
+	packaging_python39
+	pycparser_python39
+	pyparsing_python39
```

Comment:
these 2 look a bit strange - iirc these are not required by borg, are these indirect requirements?

Reply:
will do another round to check why I  added them and if they are needed